mmv1/templates/terraform/examples/gkebackup_restoreplan_all_cluster_resources.tf.erb

```diff
@@ -0,0 +1,39 @@
+resource "google_container_cluster" "primary" {
+  name               = "<%= ctx[:vars]['name'] %>-cluster"
+  location           = "us-central1"
+  initial_node_count = 1
+  workload_identity_config {
+    workload_pool = "<%= ctx[:test_env_vars]['project'] %>.svc.id.goog"
+  }
+  addons_config {
+    gke_backup_agent_config {
+      enabled = true
+    }
+  }
+}
+
+resource "google_gke_backup_backup_plan" "basic" {
+  name = "bp-<%= ctx[:vars]['name'] %>"
+  cluster = google_container_cluster.primary.id
+  location = "us-central1"
+  backup_config {
+    include_volume_data = true
+    include_secrets = true
+    all_namespaces = true
+  }
+}
+
+resource "google_gke_backup_restore_plan" "all_cluster_resources" {
+  name = "rp-<%= ctx[:vars]['name'] %>"
```

Comment:
This resource name should start with `tf-test-` and match the resource name in the yaml file.

Reply:
tf-test should be prepended by `<%= ctx[:vars]['name'] %>` automatically as long as there's a hyphen in the `name` value!

Reply:
I'm still seeing missing tests for the IAM binding. I never had to add tests for this when we added BackupPlan resource, are these tests needed?